en/flight_modes/offboard.md

```diff
@@ -35,6 +26,23 @@ Offboard mode requires an active connection to a remote MAVLink system (e.g. com
 If the connection is lost, after a timeout ([COM_OF_LOSS_T](#COM_OF_LOSS_T)) the vehicle will attempt to land or perform some other failsafe action. 
 The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL_RC_ACT](#COM_OBL_RC_ACT).
 
+## Supported Messages
+
+### Copter/VTOL
+
+* [SET_POSITION_TARGET_LOCAL_NED](https://mavlink.io/en/messages/common.html#SET_POSITION_TARGET_LOCAL_NED) - Control vehicle position, velocity, or thrust. 
+  - Acceleration setpoints are combined to create a 'thrust' setpoint.
+  - PX4 supports the coordinate frames (`coordinate_frame` field): [MAV_FRAME_LOCAL_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_LOCAL_NED) and [MAV_FRAME_BODY_NED](https://mavlink.io/en/messages/common.html#MAV_FRAME_BODY_NED).
+* [SET_ATTITUDE_TARGET](https://mavlink.io/en/messages/common.html#SET_ATTITUDE_TARGET) - Control vehicle attitude/orientation.
+
+### Fixed Wing

```

Comment:
@Jaeyoung-Lim Can you check this bit? Specifically, is there anything else we need to say about the fixed-wing support for offboard mode?

Reply:
@hamishwillee I think that looks good to me.  Would it be worth mentioning that the position setpoints are not supported, since a lot of people have been trying to use it?

Reply:
Excellent point. I have replaced the existing note with a warning about this, and will now merge.
Thanks!